incubator/consul/values.yaml

```diff
@@ -0,0 +1,208 @@
+## Bitnami Osclass image version
+## ref: https://hub.docker.com/r/bitnami/osclass/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/consul
+  tag: 1.0.6
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: Always
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistrKeySecretName
+
+## Kubernetes service type
+serviceType: ClusterIP
+
+## Consul replicas
+replicas: 3
+
+## Consul service ports
+httpPort: 8500
+rpcPort: 8400
+serflanPort: 8301
+serverPort: 8300
+consulDnsPort: 8600
+
+## Datacenter name for consul. If not supplied, will use the consul
+## default 'dc1'
+datacenterName: dc1
+
+## Predefined value for gossip key.
+## The key must be 16-bytes, can be generated with $(consul keygen)
+# gossipKey: 887Syd/BOvbtvRAKviazMg==
+
+## Use TLS to verify the authenticity of servers and clients.
+## Check README for more information.
+# tlsEncryptionSecretName: your-already-created-secret
+
+## Extra configuration that will be added to the default one.
+#localConfig: |-
+#  {
+#    "key": "value"
+#  }
+
+## Consul domain name.
+domain: consul
+
+## Consul address in which will bind client interfaces.
+clientAddress: '0.0.0.0'
```

Comment:
Should we expose these options? Shouldn't it always be `0.0.0.0` (same for `serflanAddress`)?